.codecov.yml

```diff
@@ -0,0 +1,32 @@
+coverage:
+  range: 50..100
+  round: down
+  precision: 0
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes
+        target: 50%
+        threshold: 2%
+        if_no_uploads: error
+        if_not_found: success
+        if_ci_failed: error
+
+    patch:
+      default:
+        enabled: yes
+        target: 50%
+        branches: null
+        threshold: 2%
+        if_no_uploads: error
+        if_not_found: success
+        if_ci_failed: error
+
+    changes:
+      default:
+        enabled: no
+
+comment: false
```

Comment:
I think we only need to set this to `true` to enable commenting - no special permissions are needed to allow commenting on PRs

Reply:
Ya, I'll update that. I can't even few the reports on codecov yet. I think the project is in a bit of a wonky state at the moment. We added codecov when the project was private, and then made it public. The hooks seem to still be in place though, but a few things don't work and give me errors or weird one about private scope.

I'll update this and see if the codecovs start to show up